src/test/resources/messages/common-platform/hearing-with-legal-entity-defendant.json

```diff
@@ -0,0 +1,85 @@
+{
+  "courtCentre": {
+    "id": "9b583616-049b-30f9-a14f-028a53b7cfe8",
+    "roomId": "7cb09222-49e1-3622-a5a6-ad253d2b3c39",
+    "roomName": "Crown Court 3-1",
+    "code": "B10JQ00"
+  },
+  "hearingDays": [
+    {
+      "listedDurationMinutes": 60,
+      "listingSequence": 0,
+      "sittingDay": "2021-09-08T09:00:00.000Z"
+    } ,
+    {
+      "listedDurationMinutes": 30,
+      "listingSequence": 1,
+      "sittingDay": "2021-09-09T10:30:00.000Z"
+    }
+  ],
+  "id": "8bbb4fe3-a899-45c7-bdd4-4ee25ac5a83f",
+  "jurisdictionType": "CROWN",
+  "prosecutionCases": [
+    {
+      "defendants": [
+        {
+          "id": "0ab7c3e5-eb4c-4e3f-b9e6-b9e78d3ea199",
+          "pncId": "2004/0012345U",
+          "offences": [
+            {
+              "id": "a63d9020-aa6b-4997-92fd-72a692b036de",
```

Comment:
Not related to this PR but should we start to store this ID ? It seems like we've been told we can rely on these UUIDs so might be useful. We did it for defendant and might be good to do it here.

Reply:
Yeah it's a good question, maybe we should 🤔 I'll raise a placeholder ticket for discussion